src/java/org/apache/cassandra/db/streaming/CassandraEntireSSTableStreamReader.java

```diff
@@ -53,6 +53,8 @@ public class CassandraEntireSSTableStreamReader implements IStreamReader
 {
     private static final Logger logger = LoggerFactory.getLogger(CassandraEntireSSTableStreamReader.class);
 
+    private static final boolean SKIPPED_MUTATING_STATS_AFTER_ZCS = Boolean.getBoolean("cassandra.skip_mutating_stats_after_zcs");
```

Comment:
Nit: rename to `SKIP_...` (instead of `SKIPPED_...`) for consistency with the property and because it reads a bit better.